back/api/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from dataclasses import dataclass
+import os
+import subprocess
+
+# Create your models here.
+@dataclass
+class Student:
+    username: models.CharField(null=True, blank=True, max_length = 255)
```

Comment:
Why allowing a blank username?
And why adding null = True? https://stackoverflow.com/questions/8609192/what-is-the-difference-between-null-true-and-blank-true-in-django says that CharField are never stored with Null when we are using blank = True, because that means that you can validate a form with and empty string, and then it will be stored as it and not as a Null field.